openslides/utils/settings.py.tpl

```diff
@@ -136,6 +136,8 @@ MEDIA_ROOT = os.path.join(OPENSLIDES_USER_DATA_PATH, 'media', '')
 # https://docs.djangoproject.com/en/1.10/topics/auth/passwords/#module-django.contrib.auth.password_validation
 # AUTH_PASSWORD_VALIDATORS = []
 
+# Disable the last_login field for users.
```

Comment:
We can move this settings to global settings because we don't have an existing use case at the moment to enable this field.

Reply:
Done.